Dockerfile

```diff
@@ -512,5 +512,12 @@ ADD patches/kaggle_gcp.py /root/.local/lib/python3.6/site-packages/kaggle_gcp.py
 ADD patches/kaggle_secrets.py /root/.local/lib/python3.6/site-packages/kaggle_secrets.py
 ADD patches/sitecustomize.py /root/.local/lib/python3.6/site-packages/sitecustomize.py
 
+# TensorBoard Jupyter extension
```

Comment:
Does this need to be repeated after the GPU tensorflow installation given the dependency on that?  Per the package description, I don't think it would need it.  Have you tested it?

Reply:
It may be needed once TensorBoard releases their extension. Adding.